src/components/App.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import Main from "./Main";
+import PropTypes from 'prop-types';
+
+const App = (props) => {
+  const {filmInfo} = props;
+  const {movies} = props;
```

Comment:
```suggestion
  const {filmInfo, movies} = props;
```
Декомпозицю лучше делать за один раз. два раза писать слово props - пальцы устанут